src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,11 +1,13 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
-public interface MachineProducer {
+public interface MachineProducer<T extends Machine> {
     /**
      * In your implementations - create 2-3 models, add them to the list and return
      * @return - the list of models
      */
-    List<Object> get();
+    int NUMBER_OF_MACHINES = 3;
```

Comment:
As I know,  using constants inside the interfaces is a bad practice. There is a nice explanation from the "Effective Java" book.
https://stackoverflow.com/questions/2659593/what-is-the-use-of-interface-constants